public/v4/apps/chatwoot-botpress-bridge.yml

```diff
@@ -0,0 +1,46 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: douglara/chatwoot-botpress-bridge:$$cap_version
+        restart: always
+        environment:
+            RAILS_ENV: production
+            CHATWOOT_ENDPOINT: $$cap_chatwoot_endpoint
+            CHATWOOT_BOT_TOKEN: $$cap_chatwoot_bot_token
+            BOTPRESS_ENDPOINT: $$cap_botpress_endpoint
+            BOTPRESS_BOT_ID: $$cap_botpress_bot_id
+            SECRET_KEY_BASE: $$cap_secret_key_base
+        caproverExtra:
+            containerHttpPort: '3000'
+caproverOneClickApp:
+    variables:
+        - id: $$cap_version
+          label: Chatwoot Botpress Bridge Version
+          defaultValue: '0'
```

Comment:
What's version `0`?

Reply:
Its about available app docker image, but I fixed this in app